strictdoc/backend/sdoc/writer.py

```diff
@@ -130,6 +132,37 @@ def write(self, document: Document):
                     output += requirement_in_toc
                     output += "\n"
 
+        document_view: DocumentView = document.view
+        if document_view:
+            assert document_view.default_view is not None
```

Comment:
Besides the fact that the `default_view` should go to the DefaultConfig` as per my other comment, this field should be optional and therefore this assert should be removed because we want to support the case when there is no default view specified.